models/image.py

```diff
@@ -1071,7 +1099,7 @@ def invent_filepath(self):
             try:
                 section_id_int = int(self.section_id)
             except ValueError:
-                section_id_int = 0
+                section_id_int = 0  # TODO: maybe replace with a placeholder like 99?
```

Comment:
If only there were an Int equivalent of NaN, that would be the right thing to use here.

When do we use the section_id_int?  Perhaps we should just embrace that section_id needs to be a string.

Reply:
The problem is that if you are using actual integer values (like CCD number) you'd want the (customizable) naming convention to allow you to do things like 01, 02... in the filename, and that only works with integers. 

The only problem is that same naming convention will be incorrect when `section_id=None`. I am not sure what to do about this, hence the TODO.